auto_dev/commands/augment.py

```diff
@@ -466,8 +492,37 @@ def skill_from_fsm(spec_file: str, skill_public_id: PublicId, auto_confirm: bool
 
     output = scaffolder.scaffold()
     logger.info(f"Skill scaffolded: {output}")
+    write_to_file(behaviour_path, output, FileType.PYTHON)
+    logger.info(f"Behaviours file updated: {behaviour_path}")
+
+    # Extract the FSM behavior class name
+    class_name = None
+    for line in output.splitlines():
+        if line.startswith("class ") and "FsmBehaviour" in line:
+            class_name = line.split("(")[0].replace("class ", "").strip()
+            break
+
+    if not class_name:
+        logger.error("Could not find FSM behavior class name in generated code")
+        sys.exit(1)
```

Comment:
validator will already throw, so should just use the class name form the fsm spec right?

Reply:
yeah thats what im trying to do